.github/workflows/ci.yml

```diff
@@ -78,6 +78,17 @@ jobs:
       run: |
         nix-store --import < ~/nix-cache/cache.nar
         rm -rf ~/nix-cache
+  
+    - name: Install Poetry
+      uses: snok/install-poetry@v1
+      with:
+        version: 1.5.0
+
+    - name: Configure Poetry
+      run: |
+        poetry config installer.no-binary mypy
+        poetry install
```

Comment:
Hmm this shouldn't be necessary, we install the Poetry dependencies through Nix, so using it directly will cause trouble.